integration-libs/opf/styles/components/_opf-checkout-payments.scss

```diff
@@ -0,0 +1,26 @@
+%cx-opf-checkout-payments {
+  > div {
+    border: 1px solid #d3d6db;
+    padding: 1.75rem 1.75rem;
+    border-radius: 10px;
+  }
+
+  .title {
+    font-weight: 600;
+    font-size: 18px;
+  }
+
+  .form-check {
+    margin-bottom: 0;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+
+    &:not(:first-child) {
```

Comment:
As we checked, this line won't work because there are no first child with `.form-check` in code below:
```html
<ng-container *ngIf="activeConfiguratons$ | async as activeConfiguratons">
  <div>
    <div class="title">{{ 'opf.checkout.paymentOption' | cxTranslate }}</div>
    <div class="form-check" *ngFor="let configuration of activeConfiguratons">
    ...
  </div>
</ng-container>
```

Reply:
Fixed, thanks!